src/components/InstrumentControls/Telescope.vue

```diff
@@ -98,23 +109,63 @@
           {{ frame_option }}
         </option>
       </b-select>
-    </b-field>
 
-    <b-field>
       <p class="control">
         <command-button
           :data="mount_slew_radec_command"
           style="margin-bottom: 1em"
           class="is-small is-success is-outlined"
         />
       </p>
+
+    </b-field>
+
+
+
+    <b-field horizontal label="Hour Angle">
+      <b-field>
+        <b-input
+          name="subject"
+          type="search"
+          size="is-small"
+          v-model="mount_ha"
+          autocomplete="off"
+        ></b-input>
+        <!--p class="control"><span class="button is-static is-small">deg</span></p-->
+      </b-field>
       <p class="control">
         <command-button
           :data="mount_slew_hadec_command"
           style="margin-bottom: 1em; border-left-color: #48c774"
           class="is-small"
         />
       </p>
+    </b-field>
+
+    <b-field horizontal label="Azimuth">
+      <b-field>
+        <b-input
+          name="subject"
+          type="search"
+          size="is-small"
+          v-model="mount_az"
+          autocomplete="off"
+        ></b-input>
+        <!--p class="control"><span class="button is-static is-small">hrs</span></p-->
+      </b-field>
+    </b-field>
+
+    <b-field horizontal label="Altitude">
+      <b-field>
+        <b-input
+          name="subject"
+          type="search"
+          size="is-small"
+          v-model="mount_alt"
+          autocomplete="off"
+        ></b-input>
+        <!--p class="control"><span class="button is-static is-small">deg</span></p-->
+      </b-field>
```

Comment:
Not sure whether this pull request had an effect on this or if it was a bug from earlier, but on my local instance, "slew to HA/Dec" button has one (left) side in green, similar to "slew to RA/Dec." This may have been a question settled earlier, but should both the "slew to HA/Dec" and "slew to alt/az" buttons also be green to indicate clicking on them will move the telescope? If not, might want to double check why the styling is slightly off

Reply:
This style bug is a remnant from when the two buttons were combined, and buefy/bulma was being weird with button borders The grey border on the right button was covering the green border on the left button, so I manually set a left green border on the right button to correctly render the full green border. 

Confusing, perhaps, and don't worry if the above doesn't make sense. Just remove `border-left-color: #48c774` in line 139 of this file (the element `<command-button :data="mount_slew_hadec_command" .../>`).

Reply:
Well..... hmmmz. Slew to RA/Dec is the one you'd press 99% of the time. You'd only press the other two if you were Wayne or me.... There is no reason to slew to a HA or an alt or an az unless you are doing flats and biases and things. MAYBE occasionally looking at celestial motion in the course, but the RA/Dec is the big green button realistically.... so maybe it deserves to stay green? So users know that is the big green button and the others are buttons of lesser import?

Reply:
That logic makes sense to me, Michael! It reminds me of a very similar to a design discussion Tim brought up with the camera tab - not wanting things to feel too cluttered and overwhelming (i.e. every button being big green buttons) but also showing some sort of distinction between buttons that Do Things Physically and buttons that are just on the website, which is something that's probably more of a question for the site as a whole. But definitely agree that it makes sense to make the button we want to encourage users to push the most obvious